Samples/excel-shared-runtime-global-state/assets/sample.json

```
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "office-add-in-shared-runtime-global-data",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Share global data with a shared runtime",
+      "shortDescription": "Shows how to set up a basic Office Add-in project that uses the shared runtime.",
```

Not sure it's a "basic" concept ;-)
```suggestion
      "shortDescription": "Shows how to set up an Office Add-in project that uses the shared runtime.",
```